templates/coderd.yaml

```diff
@@ -82,18 +90,26 @@ spec:
               echo Waiting on db;
               /wait_postgres.sh --host="$DB_HOST" --port="$DB_PORT" -U "$DB_USER";
               echo Starting entrypoint.sh;
-              exec /entrypoint.sh cemanager migrate up
+              exec /entrypoint.sh coderd migrate up
       containers:
-        - name: cemanager
+        - name: {{ include "coder.serviceName" . }}
+          {{- if hasKey .Values "cemanager" }}
           image: {{ .Values.cemanager.image | quote }}
+          {{- else }}
+          image: {{ .Values.coderd.image | quote }}
+          {{- end }}
           imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
           ports:
-            - name: tcp-cemanager
+            - name: tcp-{{ include "coder.serviceName" . }}
```

Comment:
do we need a port here for 8443? how about 5349?

Reply:
I've added 5349. 8443 is related to the TLS work, so I'd rather that be a separate PR.